### DISCUSSION TITLE
Help with pages build deployment

### DISCUSSION THREAD

**Discussion by krakatoa6996** [community/community#101517] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am having issues with my portfolio being published. This is my first time doing this and not familiar with the process. Can someone take a look and see if they have any ideas?

![image](https://github.com/community/community/assets/124010451/83140811-3b3e-41b6-9e37-6d3e5b3ed12a)

**Comment by fury-05** [comment]
Can you provide link to repository to look at code or error log ?

**Reply by krakatoa6996** [reply]
It appears to be resolved now. I just tried deleting the repository and restarting and that seemed to work. Thank you though!
